src/suggest-item.jsx

```diff
@@ -45,7 +57,13 @@ export default class SuggestItem extends React.Component {
       onMouseDown={this.props.onMouseDown}
       onMouseOut={this.props.onMouseOut}
       onClick={this.onClick}>
-        {this.props.suggest.label}
+        { this.props.isHighLightMatch
+          ? <span
+            dangerouslySetInnerHTML={{
+              __html: this.formatMatchedText(
+                this.props.userInput, this.props.suggest.label)
+            }}/>
```

Comment:
This looks dangerous to me. Can’t this be solved without this? Split the text in the function above in three parts: pre, match and post. And then return a regular JSX. It would be possible to render undangerous then:

```jsx
<span>
  {this.formatMatchedText(this.props.userInput, this.props.suggest.label)}
</span>
```